### ISSUE TITLE
Clearer information which targetSdkVersion is used

### ISSUE THREAD

**Issue by AlleSchonWeg** [dotnet/android#9643]
### Android framework version

Other

### Affected platform version

VS 2022

### Description

This old issue shows that for devs is not clear which targetSdkVersion a maui project target: https://github.com/dotnet/android/issues/8331
In the csproj we set something like this:
`<TargetFramework>net8.0-android</TargetFramework>`
and the compiler generates the `<uses-sdk/>` node in the manifest with a android:targetSdkVersion="xx" value. The docs: https://learn.microsoft.com/en-us/dotnet/maui/migration/android-projects?view=net-maui-9.0#changes-to-androidmanifestxml mention that Net8.0-android is shorthand for net8.0-android34.0. But also that net8.0-android35.0 is possible when a binding is available for the next Android release.

### Steps to Reproduce

The problem is we don't know when a new binding is available and with `<TargetFramework>net8.0-android</TargetFramework>` we could compile with targetSdkVersion 35 after a VS/Maui update.
We need a better info what is finally set in the manifest. It is important, because we have to test our app.

### Relevant log output

```shell

```

**Comment by jpobst** [comment]
Is your concern that in a future update `net8.0-android` may change from meaning `net8.0-android34` to `net8.0-android35`?

That will not happen, `net8.0-android` will always mean `net8.0-android34` no matter what additional level(s) are released in the future.  This is intentional as we explicitly do not want a VS/Maui update to change what Android API level an app is targeting.  

Changing the Android API level should be something that a user must manually opt into.

**Comment by AlleSchonWeg** [comment]
Hi,
yes this is my concern. I think about to change the value in our csproj file from `net8.0-android` to `net8.0-android34.0`. Because nobody in my team (except me) knows that `net8.0-android` target API 34. With this value it should be clearer to all team members which AP level is the currently target.
Can i change the API level manually to 35 with `net8.0-android35.0` in the csproj file? Is it possible atm?

**Comment by jpobst** [comment]
> Can i change the API level manually to 35 with net8.0-android35.0 in the csproj file? Is it possible atm?

No, .NET 8 only supports API-34.  To target API-35 you will need to use .NET 9.
